.github/workflows/lint.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        go-version: [1.19.x]
+        go-version: [1.19.x, 1.20.x]
```

Comment:
@morningvera do I want both versions or just latest?

Reply:
I think it’s safe to just do latest